src/components/Dropdown/Dropdown.styles.js

```diff
@@ -180,7 +182,7 @@ const dropdownIndicator =
         transition: `transform 0.1s ${getCSSVar("expand-animation-timing")}`,
         transform: selectProps.menuIsOpen ? "rotate(180deg)" : "rotate(0deg)"
       },
-      color: getCSSVar("icon-color"),
+      color: !isDisabled ? getCSSVar("icon-color") : getCSSVar("disabled-text-color"),
```

Comment:
I think it will be more clearer without the ! (more readable)
```suggestion
      color: isDisabled ? getCSSVar("disabled-text-color") : getCSSVar("icon-color"),
```